### PR TITLE
Replace MuseScore with Musescore (+ collect_artifacts)

### PR DESCRIPTION
**Pull request by anatoly-os**
In the packages titles and build docs

MuseScore was decided to be renamed to Musescore everywhere for marketing purposes. Most of the renaming process has been done on other resources, but the software is the most complex one. Let's start with visible packages names.

Next step is to replace MuseScore with Musescore in all visible strings.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -279,7 +279,7 @@ set(CMAKE_BUILD_WITH_INSTALL_RPATH ON) # Call CMake with option -DCMAKE_SKIP_RPA
 set(CMAKE_SKIP_RULE_DEPENDENCY TRUE)
 
 # The MuseScore version number.
```
Comment (Howard-C) [inline CMakeLists.txt]: This should be changed too?

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in [hunk 2]
```diff
@@ -3,13 +3,13 @@
 <component type="desktop-application">
   <id>org.musescore.MuseScore@MSCORE_INSTALL_SUFFIX@</id>
   <translation type="qt">mscore</translation>
-  <name>MuseScore</name>
+  <name>Musescore</name>
   <summary>Create, playback, and print sheet music for free</summary>
   <developer_name>MuseScore BVBA</developer_name>
```
Comment (Howard-C) [inline build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: This should be changed too?

File: build/Linux+BSD/portable/RecipeDebian [hunk 3]
```diff
@@ -126,7 +126,7 @@ fetch-package-dependencies() {
 # BUILD MuseScore PORTABLE (Compile, not package)
```
Comment (Howard-C) [inline build/Linux+BSD/portable/RecipeDebian]: This should be changed too?

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in (hunk 2 above)
Comment (anatoly-os) [reply build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: No, the company name is MuseScore BVBA.

Comment (Howard-C) [reply build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: OK.

**Pull request closed without merging**